operator/pkg/controller/workspace/tfc_org.go

```diff
@@ -67,7 +68,6 @@ func (t *TerraformCloudClient) CheckWorkspace(workspace string, instance *appv1a
 
 	return ws.ID, err
```

Comment:
I think this will return nil in the case that the workspace was created.  Can we refactor CreateWorkspace to return the Workspace object rather than just the ID?

Reply:
`ws = &tfc.Workspace{ID: id}` after a create is what prevents this. This is my first time writing Go so I could be missing something.

Reply:
oh, duh... right you are.